src/hubmap_translator.py

```diff
@@ -49,6 +49,9 @@ class Translator(TranslatorInterface):
     ACCESS_LEVEL_PUBLIC = 'public'
     ACCESS_LEVEL_CONSORTIUM = 'consortium'
     DATASET_STATUS_PUBLISHED = 'published'
+    # Base URL and constants to build endpoint URLs for talking to Ontology API
+    ONTOLOGY_API_BASE_URL = 'https://ontology-api.dev.hubmapconsortium.org'.strip('/')
```

Comment:
@kburke there still must be confusions. We'll only need the base URL configurable, that's really what meant. DEV/TEST uses the DEV instance of ontology-api and STAGE/PROD uses the PROD instance. We can't hardcode the base URL.